sympy/matrices/matrices.py

```diff
@@ -78,2091 +83,224 @@ def __repr__(self):
 
 
 class MatrixDeterminant(MatrixCommon):
-    """Provides basic matrix determinant operations.
-    Should not be instantiated directly."""
+    """Provides basic matrix determinant operations. Should not be instantiated
+    directly. See ``determinant.py`` for their implementations."""
+
+    _find_reasonable_pivot       = _find_reasonable_pivot
+    _find_reasonable_pivot_naive = _find_reasonable_pivot_naive
```

Comment:
These two are not currently methods.

Reply:
I know, @sylee957 suggested I make them methods so they could be overloaded - like by a sparse class which could implement them better due to the extra information it has about zeros.